roofit/roofitcore/src/RooDataHist.cxx

```diff
@@ -1266,67 +1271,58 @@ void RooDataHist::weightError(Double_t& lo, Double_t& hi, ErrorType etype) const
 /// Perform boundary safe 'intOrder'-th interpolation of weights in dimension 'dim'
 /// at current value 'xval'
 
```

Comment:
If you could add the documentation of the parameters here? Now is probably the best time, because you had to look at what they mean.

Reply:
Okay!